make/modules/java.base/gensrc/GensrcRegex.gmk

```diff
@@ -0,0 +1,46 @@
+#
+# Copyright (c) 2023, Oracle and/or its affiliates. All rights reserved.
+# DO NOT ALTER OR REMOVE COPYRIGHT NOTICES OR THIS FILE HEADER.
+#
+# This code is free software; you can redistribute it and/or modify it
+# under the terms of the GNU General Public License version 2 only, as
+# published by the Free Software Foundation.  Oracle designates this
+# particular file as subject to the "Classpath" exception as provided
+# by Oracle in the LICENSE file that accompanied this code.
+#
+# This code is distributed in the hope that it will be useful, but WITHOUT
+# ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or
+# FITNESS FOR A PARTICULAR PURPOSE.  See the GNU General Public License
+# version 2 for more details (a copy is included in the LICENSE file that
+# accompanied this code).
+#
+# You should have received a copy of the GNU General Public License version
+# 2 along with this work; if not, write to the Free Software Foundation,
+# Inc., 51 Franklin St, Fifth Floor, Boston, MA 02110-1301 USA.
+#
+# Please contact Oracle, 500 Oracle Parkway, Redwood Shores, CA 94065 USA
+# or visit www.oracle.com if you need additional information or have any
+# questions.
+#
+
+#
+# Rules to create java files under
+# $(SUPPORT_OUTPUTDIR)/gensrc/java.base/jdk/internal/util/regex/
+#
+
+GENSRC_INDICCONJUNCTBREAK := $(SUPPORT_OUTPUTDIR)/gensrc/java.base/jdk/internal/util/regex/IndicConjunctBreak.java
+
+INDICCONJUNCTBREAKTEMP = $(MODULE_SRC)/share/classes/jdk/internal/util/regex/IndicConjunctBreak.java.template
+INDICCONJUNCTBREAKPROPS = $(MODULE_SRC)/share/data/unicodedata/DerivedCoreProperties.txt
+INDICCONJUNCTBREAKPARAMS = InCB\=Linker InCB\=Extend InCB\=Consonant
+
+$(GENSRC_INDICCONJUNCTBREAK): $(BUILD_TOOLS_JDK) $(INDICCONJUNCTBREAKTEMP) $(INDICCONJUNCTBREAKPROPS)
+	$(call LogInfo, Generating $@)
+	$(call MakeTargetDir)
+	$(TOOL_GENERATEEXTRAPROPERTIES) \
+		$(INDICCONJUNCTBREAKTEMP) \
+		$(INDICCONJUNCTBREAKPROPS) \
+		$(GENSRC_INDICCONJUNCTBREAK) \
+		$(INDICCONJUNCTBREAKPARAMS)
```

Comment:
Please use 4 spaces after the initial tab for continuation indentation in recipes (see point 2 and 6 https://openjdk.org/groups/build/doc/code-conventions.html).

Reply:
I wasn't aware of the convention (initial tab, then spaces for further indentation). Thanks.